README.md

```diff
@@ -1,2 +1,36 @@
 # nf-agora
-Nextflow workflow for agora-data-tools data processing
+
+## Intro
+
+This repository contains a Nextflow Pipeline that wraps agora-data-tools and therefore the Agora ETL process. It is deployed on Sage's Nextflow Tower Instance, making it simple to trigger and monitor runs. 
+
+## Running the pipeline on Nextflow Tower
+
+### Getting Oriented in Nextflow Tower
+
+1. Navigate to https://tower.sagebionetworks.org/login.
+2. Click “Sign in with Google” and enter your Sage credentials.
+3. In the top left corner, you should see “Sage-Bionetworks/agora-project”. This indicates that you are in the workspace for Agora in our Nextflow Tower instance.
```

Comment:
Nit: We may want to add a screenshot here on whats expected.  For instance, it's important that the "docker" profile is not removed